Alignment/OfflineValidation/test/examples/PV_alignments_different_datasets.yaml

```diff
@@ -29,8 +29,7 @@ validations:
                 singles:
                 - 2016postVFPMC
                 - 2016preVFPMC
-                - 2017MC
-                - 2018MC
+                - 2017And2018MC
```

Comment:
Is it really the same scenario? or is it just for the test?

Reply:
It's just for a test and aren't correct physics wise. The examples aren't ideal and can be improved but I don't have time until I hand in my thesis at the end of March. I still think they serve as reasonable examples for how to set up different configs for now.